src/actions/index.js

```diff
@@ -103,10 +103,10 @@ export const fetchStorageData = client => {
         offersLink = `${managerUrl}/cozy/instances/${uuid}/premium`
       }
     } catch (e) {
-      if (e.error && e.error !== 'Not Found') {
+      if (e.reason.error && e.reason.error !== 'Not Found') {
         // eslint-disable-next-line no-console
         console.warn(e)
-      } else if (!e.error) {
+      } else if (!e.reason.error) {
```

Comment:
I don't understand that condition.

As we do exactly the same thing on both "if" and "else if" block, can this be simplified by this?
```js
if (e.reason.error !== 'Not Found') {
  console.warn(e)
}
```

Also do you know the reason why we don't log anything in case of 'Not Found'?

Reply:
Me either but we could simplify it by returning `console.warn(e)` directly to see if there is a 404 problem 🤔

Reply:
I think we should simplify it 👍